db/db_block_cache_test.cc

```diff
@@ -1870,8 +1871,13 @@ TEST_F(CacheKeyTest, Encodings) {
     }
     db_id_ = r.Next();
 
-    CacheKey last_base;
+#ifndef __clang_analyzer__
+    CacheKey last_base;  // Clang analyzer thinks this is garbage
     ASSERT_TRUE(last_base.IsEmpty());
+#else
+    // Work-around
+    CacheKey last_base = CacheKey::CreateUniqueForProcessLifetime();
```

Comment:
If this is a suitable workaround for the analyzer, is there a reason not to just make it the standard code (rather than conditional)?  If there is, please add a comment as to why the difference is a good thing.

Reply:
I wanted to preserve the intended original code, but I've now thought of a way to do that without conditional compilation.

Reply:
Clang-analyze leaves me with few options, weirdly